src/views/common/list/GenericDraggableList/index.tsx

```diff
@@ -36,7 +37,10 @@ export default function GenericDraggableList<ColumnNames>({
 
     return (
         <DragDropContext onDragEnd={onDragEnd}>
-            <Droppable droppableId="droppable">
+            <Droppable
+                droppableId="droppable"
+                isDropDisabled={!checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC')}
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'